### PR TITLE
the mongoose dependency have been added to connect to mongodb

### PR DESCRIPTION
**Pull request by Imaraika**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +1,5 @@
 my_super_secret_key= 9yhggtt*;
+
+ "MONGO_ATLAS_PW": "ange-admin123456";
+
+
```
Comment (Anguandia) [inline .env:5]: Would you consider using the environment for storing the secrete key?

**Review by maximeish — COMMENTED** [review]
the  .env file should be not be public. 
You can remove it locally from the cache so that it doesn't get pushed by **git rm --cached .env**

File: SRC/controllers/postsContr.js [hunk 2]
```diff
@@ -18,8 +21,22 @@ export const createPost = (req, res) => {
...
+    const product = new Product({
+        // _id:new mongoose.Types.ObjectId(),
+        title: req.body.title,
+        content: req.body.content,
+        author: req.body.author
+    });
+    product
+    .save()
+    .then(result => {
+        console.log(result);
```
Comment (Anguandia) [inline SRC/controllers/postsContr.js:33]: We don't need to console log outside the development environment

File: nodemon.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+    "env":
+    { 
+        "MONGO_ATLAS_PW": "ange-admin123456"
```
Comment (julesntare) [inline nodemon.json:4]: remove this file of nodemon because it contains your secrets, try to add secrets in .env

File: SRC/index.js [hunk 4]
```diff
@@ -10,7 +10,13 @@ import routes from './routesTesting/post.routes.test';
 import routeMsgTst from './routesTesting/message.routes.test';
 import routeCmtTst from './routesTesting/comments.routes.test';
 import routeUserTst from './routesTesting/users.route.test';
+import dotEnv from 'dotenv';
+dotEnv.config() 
 
+const mongoose = require('mongoose');
```
Comment (Anguandia) [inline SRC/index.js:16]: I'll look through the entire codebase and in our one-on-one, well guide on refactoring to eliminate duplication.

**Comment by julesntare** [comment]
I think also the coverages and nyc_output don't have to be pushed. add those folders in .gitignore

**Review by Anguandia — COMMENTED** [review]
Do not merge yet, I'll take a closer look and we discuss during the one-on-one, sometime later today

**Comment by maximeish** [comment]
> I think also the coverages and nyc_output don't have to be pushed. add those folders in .gitignore

Yes and also clear them from the cache by `git rm --cached coverage/ .nyc_output/`

**Review by Anguandia — COMMENTED** [review]
Please consider having smaller pull requests with fewer changes next time

**Pull request merged**